packages/theme/src/Components/TextField/index.tsx

```diff
@@ -48,7 +48,7 @@ const useStyles = makeStyles()((theme) => ({
     },
 }))
 
-type MaskTextFieldProps = Exclude<StandardTextFieldProps, 'variant'>
+export type MaskTextFieldProps = Exclude<StandardTextFieldProps, 'variant'>
```

Comment:
why add the `Mask` prefix string?

Reply:
TextField is belong to material, Add `mask` can help distinguish.